website/docs/pact_broker/advanced_topics/consumer_version_selectors.md

```diff
@@ -1180,18 +1180,18 @@ verifier = Verifier(
   <TabItem value="golang">
 
 ```golang
- pact.VerifyProvider(t, types.VerifyRequest{
-    ConsumerVersionSelectors: []types.ConsumerVersionSelector{
-   types.ConsumerVersionSelector{
+ pact.VerifyProvider(t, provider.VerifyRequest{
+    ConsumerVersionSelectors: []provider.Selector{
```

Comment:
Is this meant to be `[]provider.ConsumerVersionSelector`?

Reply:
Hi according to the latest v2.0.7 version the type was changed to `type Selector interface` [here](https://github.com/pact-foundation/pact-go/blob/07d2009ddd2e4be8a7cdca6672359bde9e334a3e/provider/consumer_version_selector.go#L35)

The `provider.VerifyRequest` type expect a [Selector](https://github.com/pact-foundation/pact-go/blob/6b525b40d8937d9b37bb1844d76df23675f40aba/provider/verify_request.go#L73) interface now.

Reply:
awesome, thanks for clarifying (I was on mobile before, and it jumped out to me).